test/shm_size.bats

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+@test "check /dev/shm is changed" {
+	start_crio
+	replace_config "enable_custom_shm_size" "true"
+	restart_crio
+
+	run crictl runp "$TESTDATA"/sandbox_config_shmsize.json
```

Comment:
instead of a special pod config, can we use jq to generate the one you have below? you can do so like so:
`cat test/testdata/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"'`
then you can write that to a file in `$TESTDIR` so it's cleaned up after the test